### PR TITLE
logutil: add Slow

### PR DESCRIPTION
**Pull request by reusee**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:
Add Slow function to log slow operations

### CODE REVIEW

**Comment by matrix-meow** [comment]
@reusee Thanks for your contributions!

The pull request adds a new function called `Slow` to the `logutil` package. The function logs a message if an operation takes longer than a specified timeout duration. The changes include adding the `Slow` function to the `slow.go` file and adding a test for the function in the `slow_test.go` file.

There are no major issues with the pull request. However, there are a few minor suggestions to improve the code:

1. The copyright year in the header should be updated to the current year.
2. The function name `Slow` could be more descriptive. For example, `LogIfSlow` or `LogSlowOperation` would be more self-explanatory.
3. The test case could be improved by adding more scenarios to test the function's behavior. For example, testing the function with a timeout duration of zero or a negative value.

Other than these minor suggestions, the pull request looks good and can be merged.

File: pkg/logutil/slow.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import (
+	"sync/atomic"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// Slow logs if operation not done in timeout duration
+func Slow(timeout time.Duration, msg string, fields ...zap.Field) (doneFunc func() bool) {
+	state := new(atomic.Int32) // 0: pending, 1: logged, 2: do not log
```
Comment (fengttt) [inline pkg/logutil/slow.go:26]: do you need new?   will var state atomic.Int32 work?

**Review by fengttt — COMMENTED** [review]
Why don't we use Span?   Given that span is not logging anything if it is fast.

**Review by daviszhen — APPROVED** [review]

File: pkg/logutil/slow.go (hunk 1 above)
Comment (reusee) [reply pkg/logutil/slow.go:26]: It's the same.

**Pull request merged**